bigtop_toolchain/manifests/packages.pp

```diff
@@ -70,7 +70,7 @@
         "yasm"
       ]
 
-      if (/redhat|centos|rocky/ in downcase($operatingsystem) and Integer($operatingsystemmajrelease) >= 9) {
+      if (0 == versioncmp($operatingsystemmajrelease, '9')) {
```

Comment:
`<=` may be better than `==` considering the future release of new version.

Reply:
I refactored the conditional a bit.